atd-vzd/views/view_fatalities.sql

```diff
@@ -36,12 +36,14 @@ CREATE OR REPLACE VIEW view_fatalities AS (
             ORDER BY crashes.crash_date ASC, crashes.crash_time ASC, crashes.crash_id) 
             AS ytd_fatal_crash,
         crashes.case_id,
-        crashes.law_enforcement_num
+        crashes.law_enforcement_num,
+        engineering_areas.label as engineering_area
     FROM
         fatalities f
     INNER JOIN atd_txdot_crashes crashes ON f.crash_id = crashes.crash_id
     LEFT JOIN atd_txdot_primaryperson primaryperson ON f.primaryperson_id = primaryperson.primaryperson_id
     LEFT JOIN atd_txdot_person person ON f.person_id = person.person_id
+    LEFT JOIN engineering_areas ON ST_CONTAINS(engineering_areas.geometry, crashes.position)
```

Comment:
Because this is a view, the `engineering_area` will automatically update on edits to the crash position through the edit coordinates functionality in the VZE. I toyed with some other solutions but I personally like this one. Another way would have been to have this be a static column in the `atd_txdot_crashes`  table or `fatalities` table (which includes fatalities outside of AFP that are not included in the FTT Index) that is updated by database triggers when the crash coordinates change. But since engineering areas are only relevant to AFP fatalities and the whole point of having them in the DB is bc mary uses them when sending out messages for the Fatality Review Board, i thought it would be most clean and simple to have it be calculated on the fly when the view for the Fatalities page is created. If people have other opinions i am open to hearing them!

Reply:
I definitely like calculating on-the-fly so long as you don't run into performance issues. because the table has relatively view rows this seems like a great solution.

Reply:
Yeah, ive tested loading all of the fatalities dating back to 2013 in the VZE fatalities page and it doesnt seem to be loading any slower than before (at least from a human perspective).

Reply:
mhmm yeah i know, which frank has mentioned that `st_contains` can be an expensive calculation, so thats definitely something i was considering performance-wise and thats why i want to open up discussion incase anyone thinks doing it this way is too performance-heavy

Reply:
all good! i'd say leave it as-is until it becomes a problem 🚢